src/app/stack/utils/report-summary-utils.ts

```diff
@@ -1,3 +1,5 @@
+import _ from 'lodash';
```

Comment:
@arajkumar i know i recommended to use `lodash` but it's used just 2 places , check once if it;s worth adding lodash to bundle as it'll increase bundle size and performance will be impacted.

Reply:
Thanks @invincibleJai. Here are the size of final webpacks with & without lodash.

Without: 23792 KB
With lodash: 24320 KB
diff: 528 KB